elements/pfe-card/src/pfe-card.scss

```diff
@@ -147,59 +151,59 @@ $LOCAL: card;
 }
 
 ::slotted([slot="pfe-card--header"]) {
-  display: block;
   z-index: 2;
   // Declare the header background color
-  background-color: #{pfe-local($cssvar: BackgroundColor, $region: header)};
-  color: #{pfe-local($cssvar: Color, $region: header)};
+  background-color: pfe-local($cssvar: BackgroundColor, $region: header);
+  color:            pfe-local($cssvar: Color, $region: header);
 
   // Margin style only on top and sides
   margin-top:    -2rem; //IE11 fallback
-  margin-top:    calc(-1 * #{pfe-local(PaddingTop)}) !important;
+  margin-top:    calc(#{pfe-local(PaddingTop)} * -1) !important;
   margin-right:  -2rem; //IE11 fallback
-  margin-right:  calc(-1 * #{pfe-local(PaddingRight)});
-  margin-bottom: #{pfe-local(spacing)};
+  margin-right:  calc(#{pfe-local(PaddingRight)} * -1);
+  margin-bottom: pfe-local(spacing--vertical);
   margin-left:   -2rem; //IE11 fallback
-  margin-left:   calc(-1 * #{pfe-local(PaddingLeft)});
+  margin-left:   calc(#{pfe-local(PaddingLeft)} * -1);
 
   // Padding for the header region
-  padding-top:    #{pfe-local(spacing)}; // Top padding differs if background color applied
-  padding-right:  #{pfe-local(PaddingRight)};
-  padding-left:   #{pfe-local(PaddingLeft)};
-  padding-bottom: #{pfe-local(spacing)};
+  padding-top:    pfe-local(spacing--vertical); // Top padding differs if background color applied
+  padding-right:  pfe-local(PaddingRight);
+  padding-left:   pfe-local(PaddingLeft);
+  padding-bottom: pfe-local(spacing--vertical);
 }
 
 ::slotted(:not([slot])) {
```

Comment:
I removed the display: block from the other slots because it was too opinionated about it's children.  It was causing a break where if you added unwrapped content into the body region, it would line break any internal styling:
```
This text is unwrapped <a href="#">with a link</a>.
```

^ In that example, the period and link were on new lines as block elements.

Reply:
Yes, I saw this earlier. Good idea.